src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceSourceConfig.java

```diff
@@ -90,7 +90,7 @@ public class SalesforceSourceConfig extends SalesforceBaseSourceConfig {
   private final String parent;
 
   @VisibleForTesting
-  SalesforceSourceConfig(String referenceName,
+  public SalesforceSourceConfig(String referenceName,
```

Comment:
if this function is public for more than just being visible for testing, we should remove the `@VisibleForTesting` annotation